addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -8527,14 +8527,31 @@ bool UseAbility(const char[] ability_name, const char[] plugin_name, int boss, i
 		int button;
 		switch(buttonMode)
 		{
+		case 1:
+			{
+				button=IN_DUCK|IN_ATTACK2;
+				bossHasReloadAbility[boss]=true;
```

Comment:
Should be `bossHasRightMouseAbility[boss]=true;`

Reply:
Dammit